.github/workflows/android-app.yml

```diff
@@ -35,6 +35,13 @@ jobs:
           toolchain:    stable
           override:     true
 
+      - name:                 Install uniffi_bindgen
+        uses:                 actions-rs/install@v0.1
+        with:
+          crate:              uniffi_bindgen
+          version:            0.17.0 # Must be in sync with version in Cargo.toml
```

Comment:
Another important thing about this PR - we should update "how to build Signer" page of manual - what prerequisites are, what should be pre-installed, and that uniffi_bindgen should match Cargo.toml.